tests/provisioner_with_invalid_backend_sc.go

```diff
@@ -89,57 +86,27 @@ var _ = Describe("TEST NFS PROVISIONER WITH INVALID BACKEND SC", func() {
 				WithStorageClass(scName).
 				WithAccessModes(accessModes).
 				WithCapacity(capacity).Build()
-			Expect(err).To(BeNil(), "while building pvc {%s} in namespace {%s}", pvcName, applicationNamespace)
+			Expect(err).To(BeNil(), "while building pvc object for %s/%s", applicationNamespace, pvcName)
 
 			By("creating above pvc")
 			err = Client.createPVC(pvcObj)
-			Expect(err).To(BeNil(), "while creating pvc {%s} in namespace {%s}", pvcName, applicationNamespace)
-
-			pvcPhase, err := Client.waitForPVCBound(applicationNamespace, pvcName)
-			Expect(err).To(BeNil(), "while waiting for pvc %s/%s bound phase", applicationNamespace, pvcName)
-			Expect(pvcPhase).To(Equal(corev1.ClaimBound), "pvc %s/%s should be in bound phase", applicationNamespace, pvcName)
-		})
-	})
-
-	When("verifying nfs-server state", func() {
-		It("should have nfs-server in pending state", func() {
```

Comment:
Q: Event with the changes nfs-server deployment will be created right? Why do we need to remove this?

Reply:
Due to a timing issue. The same behavior is added in `TEST GARBAGE COLLECTION OF NFS RESOURCES`.